omero/developers/Server/LightAdmins.txt

```diff
@@ -0,0 +1,244 @@
+The server's view of administrator restrictions
+===============================================
+
+OMERO 5.4 introduced the concept of a restricted administrator. These
+are generally more powerful than normal OMERO users and group owners but
+they do not have all the powers of full administrators such as ``root``
+because they are restricted in specific ways by their
+``Experimenter.config`` property. For that property, a name of
+``AdminPrivilege:Dance`` (just an example!) with a value of ``false``
+indicates that the user's ability to dance is not that of a full
+administrator but is instead that of a normal user who is not a member
+of the ``system`` group. A member of the ``system`` group with an empty
+``config`` property is a full administrator without restrictions.
+
+
+Restrictions
+------------
+
+The :source:`meta.ome.xml mapping
+<components/model/resources/mappings/meta.ome.xml>` lists all the values
+of the :javadoc:`AdminPrivilege
+<slice2html/omero/model/AdminPrivilege.html>` enumeration. If an
+administrator has any of the below restrictions then their ability to
+perform the corresponding operation does not take account of that they
+are an administrator. It instead becomes as if they were but a normal
```

Comment:
This point "If an administrator has any of the below....." , was a little difficult to understand. Its formulated very well, but makes me stop and think about it a couple of times, but it would be great if this line could be broken down into multiple sentences.